doc/proxy-clients.rst

```diff
@@ -184,6 +184,42 @@ dispatcher as explained above and pass it to the Symfony client::
 
     $symfony = new Symfony($httpDispatcher);
 
+
+The `HttpDispatcher` sends real HTTP requests using any instance of `HttpAsyncClient`
+available in your application (see HTTPlug_ for more information). If you
+have a single node Symfony application setup you might want to directly route
+these requests through the kernel directly instead of having it execute real
+HTTP requests for e.g. performance reasons or just because it seems easier to
+setup as you don't have to know the domain/ip your application is reachable at.
```

Comment:
This makes your setup easier as you don't need to know the IP of your server and will also save server resources.